python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py

```diff
@@ -123,18 +123,25 @@ def optimize_llm(
         modeling_module_name = model.__class__.__module__
         module = importlib.import_module(modeling_module_name)
 
+        if model.config.num_hidden_layers == 52:
+            # for minicpm-1b
+            transpose_cache = transpose_value_cache
+        elif model.config.num_hidden_layers == 40:
+            # for minicpm-2b
+            transpose_cache = False
```

Comment:
Will it work on previous driver with `transpose_cache=true`?

Reply:
it could work.

Reply:
it could work with previous driver on mtl, but cannot work on lnl.